package.json

```diff
@@ -117,14 +119,14 @@
 	"scripts": {
 		"build": "tsc && npm run download:files && npm run commit:hash && cp -r public dist",
 		"commit:hash": "git rev-parse HEAD > data/LAST_API_COMMIT_HASH.txt",
-		"coverage": "npm run clean && NODE_ENV=test TEST_MODE=unit NEW_RELIC_ENABLED=false c8 mocha",
+		"coverage": "npm run clean && NODE_ENV=test TEST_MODE=unit c8 mocha",
 		"clean": "rimraf coverage",
 		"download:files": "tsx src/lib/download-files.ts",
 		"prepare": "npm run download:files; husky || echo 'Failed to install husky'",
-		"stats": "NEW_RELIC_ENABLED=false tsx probes-stats/known.ts",
-		"start": "node --max_old_space_size=3584 --max-semi-space-size=128 --experimental-loader newrelic/esm-loader.mjs -r newrelic dist/src/index.js",
-		"start:dev": "NODE_ENV=development FAKE_PROBE_IP=1 NEW_RELIC_ENABLED=false tsx src/index.ts",
-		"start:test": "NODE_ENV=test knex migrate:latest && NODE_ENV=test NEW_RELIC_ENABLED=false tsx src/index.ts",
+		"stats": "tsx probes-stats/known.ts",
+		"start": "ELASTIC_APM_CONFIG_FILE=elastic-apm-node.cjs node --max_old_space_size=3584 --max-semi-space-size=128 --experimental-loader elastic-apm-node/loader.mjs -r elastic-apm-node/start.js dist/src/index.js",
+		"start:dev": "NODE_ENV=development FAKE_PROBE_IP=1 ELASTIC_APM_CONFIG_FILE=elastic-apm-node.cjs tsx src/index.ts",
+		"start:test": "NODE_ENV=test knex migrate:latest && NODE_ENV=test ELASTIC_APM_CONFIG_FILE=elastic-apm-node.cjs tsx src/index.ts",
```

Comment:
Is `ELASTIC_APM_CONFIG_FILE=elastic-apm-node.cjs` required here?

Reply:
Yes because of the extension, it expects `js` otherwise.